app/models/wizard/steps/authenticate.rb

```diff
@@ -23,6 +24,12 @@ def save
         super
       end
 
+      def export
+        return {} if skipped?
```

Comment:
I'm looking at this and re-considering my original implementation

Originally I'd got the Wizard stripped out skipped steps, and later changed it to be steps which were skipped instead returning empty data. Now starting to rethink this should have been how I'd got it the first time.

Not an issue for this PR but may switch that back at some point

Reply:
Yeah I had the same conundrum when I implemented the review answers methods in the TTA app and opted for the wizard to call `skipped?`; otherwise I was repeating the same logic in most of the steps as most overrode the method